gooddata-sdk/gooddata_sdk/catalog_model/workspace_content_objects/__init__.py

```diff
@@ -0,0 +1,9 @@
+# (C) 2022 GoodData Corporation
+
+from gooddata_sdk.catalog_model.workspace_content_objects.dataset import (
+    CatalogAttribute,
+    CatalogDataset,
+    CatalogFact,
+    CatalogLabel,
+)
+from gooddata_sdk.catalog_model.workspace_content_objects.metric import CatalogMetric
```

Comment:
Any benefit of having this imports?

Reply:
completely refactored, inner imports removed